src/libraries/Microsoft.PowerFx.Connectors/Tabular/RelationShips/Relationships.cs

```diff
@@ -7,7 +7,7 @@
 namespace Microsoft.PowerFx.Connectors
 {
     // x-ms-relationships
-    internal sealed class Relationship
+    public sealed class Relationship
```

Comment:
>public [](http://example.com/codeflow?start=4&length=6)

Do these need to be public?

Reply:
yes, used in CdpService, CdpTable, CdpTableValue...

Reply:
What I'm getting at is that CDP shouldn't be special  - we don't want DV writing directly against CDP APIs ... we want them writing against generic Power Fx APIs that abstract out Dv/CDP/ServiceNow/etc.

Reply:
DV shouldn't be using this class
This is for ServiceNow and other connectors